### DISCUSSION TITLE
[A-L] (3 Aprile 2023) Esercizio 6

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#385] [Esami A-L]
La seguente formula proposizionale in CNF è soddisfacibile?

$$
    \Large \\{\ \\{p,\ q,\ r\\},\ \\{p,\ q,\ \neg r\\},\ \\{p, \neg q,\ r\\},\ \\{p, \neg q, \neg r\\},\ \\{\neg p,\ q,\ r\\},\
$$

$$
    \Large \\{\neg p,\ q,\ \neg r\\},\  \\{\neg p,\ \neg q,\ r\\},\ \\{\neg p,\ \neg q,\ \neg r\\}\ \\}.
$$

Se si risponde SI definire un assegnamento che la soddisfa, se si risponde NO dimostrare l’insoddisfacibilità usando la regola di Risoluzione.

**Comment by cek753** [comment]
Sat p=0 q=1 r=0
